site/app/views/GlobalView.php

```diff
@@ -52,7 +52,8 @@ public function header($breadcrumbs, $wrapper_urls, $sidebar_buttons, $notificat
             "system_message" => $this->core->getConfig()->getSystemMessage(),
             "csrf_token" => $this->core->getCsrfToken(),
             "enable_banner" => $config_data['duck_special_effects'],
-            "duck_img" => $duck_img
+            "duck_img" => $duck_img,
+            "use_mobile_viewport" => $use_mobile_viewport
```

Comment:
To follow-up from above, get rid of the `$use_mobile_viewport` from above, and have this just be `"use_mobile_viewport" => $this->output->useMobileViewport()`